example.yml

```diff
@@ -6,7 +6,9 @@ global:
   # Set this to True to skip verification of server SSL certificates.  This might
   # be useful if your Jira / Bitbucket server doesn't have a valid SSL certificate.
   no_verify_ssl: False
-
+  # Set this to False if you would not like to send your config.yml file to Jellyfish.
+  # We use this as a diagnostic tool for customer assistance.
+  send_agent_config_in_payload: True
```

Comment:
this should default to `False` here, we would want customers to toggle it on in specific diagnostic situations

Reply:
Sure, reason I had it set to `True` was because of this comment in the ticket's description:

> Let's add a configuration option to the agent called "send_agent_config_in_payload", which can be set to True or False. Let's default it to True in the example config.yml and the agent's setup instructions, with a comment indicating that the customer can flip it to False if their config.yml contains any sensitive information (but that this is uncommon).

Reply:
Ah I hadn't read that. IMO this is safer, but @mattklein if you feel strongly about it I could be convinced otherwise

Reply:
I do -- reasoning is farther down in the Jira ticket